src/CMakeLists.txt

```diff
@@ -77,24 +91,13 @@ add_dependencies(clm csm_share esmf)
 # We need to look for header files here, in order to pick up shr_assert.h
 include_directories(${CLM_ROOT}/share/include)
 
-
```

Comment:
Happy, to see the explicit link with PIO go. But, I think we should have some notes about needing to use an ESMF version that includes PIO pre-built as part of it.